apps/mull-api/src/app/channel/channel.resolver.ts

```diff
@@ -1,20 +1,27 @@
 import { UseGuards } from '@nestjs/common';
 import { Args, Int, Mutation, Query, Resolver } from '@nestjs/graphql';
 import { AuthGuard } from '../auth/auth.guard';
-import { Channel } from '../entities';
+import { Channel, DirectMessageChannel, EventChannel } from '../entities';
+import { UserInput } from '../user/inputs/user.input';
 import { ChannelService } from './channel.service';
-import { CreateChannelInput } from './inputs/channel.input';
+import { CreateDmChannelInput, CreateEventChannelInput } from './inputs/channel.input';
 @Resolver('Channel')
 export class ChannelResolver {
   constructor(private readonly channelService: ChannelService) {}
 
   @Mutation(/* istanbul ignore next */ () => Boolean)
   @UseGuards(AuthGuard)
-  async createChannel(
-    @Args('input') input: CreateChannelInput,
+  async createEventChannel(
+    @Args('input') input: CreateEventChannelInput,
     @Args('eventId', { type: /* istanbul ignore next */ () => Int }) eventId?: number
```

Comment:
`eventId` is mandatory now, we should remove the second optional argument here and have `eventId` as part of `CreateEventChannelInput`

Reply:
Done

Reply:
Upon further inspection, createEventChannel does not need to exposed in their respective service/resolver files as createEvent is the one creating those channels through `cascade: true`. Therefore, we will be removing these functions as they are not used anywhere else.